python/sglang/srt/server_args.py

```diff
@@ -169,9 +169,10 @@ def __post_init__(self):
             gpu_mem = get_amdgpu_memory_capacity()
         else:
             gpu_mem = get_nvgpu_memory_capacity()
-        if gpu_mem < 25000:
+
+        # If the GPU memory is less than 25GB (like GTX 4090) and the user hasn't manually specified the chunked prefill size, we reduce its default value by a factor of 4.
```

Comment:
When using a GTX 4090, users typically set the chunked prefill size to a smaller value themselves. In this case, applying another hidden division by 4 is unreasonable. If the default value is 8192, then this adjustment could make sense. However, setting `cuda_graph_max_bs` to 4 is also unreasonable, as we sometimes have batch sizes approaching 80 when serving on a 4090. I removed `cuda_graph_max_bs` setting.

Reply:
@BBuf Is it better to change `== 8192` to `>= some_threshold`?

Reply:
The 8192 is the serve_args `chunked_prefill_size` default value, I think if user specify a custom `chunked_prefill_size` we should not to change it.For example I use `chunked_prefill_size=256` when serving llama7b in GTX 4090, if change it to `256 // 4`, I had bad performance, but I don't know reason without profiling. So I set `chunked_prefill_size` to default value(8192).

Reply:
I think we can really leave the adjustment to `chunked_prefill_size` out for now, cause no good hypothesis yet, user can set explicitly to 8192, and models can be smaller too, so we don't need to adjust it proactively, instead we can provide a message to user for warning.